docker-compose.yml

```
@@ -0,0 +1,49 @@
+version: '3.8'
```

```suggestion
version: '3.7'
```

I couldn't build it in ubuntu focal, since version 3.8 was introduced with docker-compose [1.25.5](https://github.com/docker/compose/releases/tag/1.25.5) and the latest docker-compose in focal is [1.25.0](https://packages.ubuntu.com/search?suite=focal&arch=any&searchon=names&keywords=docker%20compose), maybe we should downgrade it even further to support bionic.